### PR TITLE
Add nparray case for basic operator

### PR DESCRIPTION
**Pull request by Charles-hit**
### PR types

Bug fixes 
### PR changes

Others
### Describe

add nparray case for basic operator.

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/test_matmul_v2_op.py [hunk 1]
```diff
@@ -714,6 +714,28 @@ def init_grad_input_output(self):
...
+
+        np_a = np.random.random((2, 4)).astype(np.float32)
+        np_b = np.random.random((4, 2)).astype(np.float32)
+
+        tensor_a = paddle.to_tensor(np_a, dtype="float32")
+        tensor_b = paddle.to_tensor(np_b, dtype="float32")
+
+        # normal case: tensor * nparray
+        expect_out = np_a * np_b
+        actual_out = tensor_a * np_b
```
Comment (veyron95) [inline python/paddle/fluid/tests/unittests/test_matmul_v2_op.py]: `* `符号不是 matmul, `@`才是 matmul

**Review by JiabinYang — APPROVED** [review]
LGTM

**Review by veyron95 — APPROVED** [review]
LGTM

File: python/paddle/fluid/tests/unittests/test_matmul_v2_op.py (hunk 1 above)
Comment (Charles-hit) [reply python/paddle/fluid/tests/unittests/test_matmul_v2_op.py]: done

**Pull request merged**